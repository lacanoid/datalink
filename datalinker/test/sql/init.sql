create extension datalinker cascade;

set search_path=datalink;

\dx datalinker

\dx+ datalinker

