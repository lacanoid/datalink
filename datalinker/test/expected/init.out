create extension datalinker cascade;
NOTICE:  installing required extension "uuid-ossp"
NOTICE:  installing required extension "plperlu"
NOTICE:  installing required extension "datalink"
set search_path=datalink;
\dx datalinker
             List of installed extensions
    Name    | Version |  Schema  |    Description     
------------+---------+----------+--------------------
 datalinker | 0.2     | datalink | SQL/MED Datalinker
(1 row)

\dx+ datalinker
Objects in extension "datalinker"
       Object description       
--------------------------------
 function dl_inode(file_path)
 function dl_paths(file_path)
 function dl_space(file_path)
 sequence dl_space_space_id_seq
 table dl_inode
 table dl_space
 type file_path
(7 rows)

